canonical-websites/www.ubuntu.com
# Update Japan legal page

## Done

Update copy of Japan legal page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja>
- Check page matches the [copy doc](https://docs.google.com/document/d/1LtJa74IBi6MLORJJLgfDrQkfDWU2YFvKhLhITQw_qNA/edit#heading=h.gjdgxs)


## Issue / Card

Fixes [#484](https://github.com/ubuntudesign/web-squad/issues/484)
